### ISSUE TITLE
ComboBox A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#6920]
Tracking all of the issues brought up with the 10/30 accessibility pass on ComboBox  control
Details on individual bugs can be found on [Updated Spreadsheet with Jeremy and Arvind's notes](https://microsoft-my.sharepoint-df.com/:x:/p/mgodbolt/EWp6C5th0sZMgOON89z0g4kBPi12LZ_vfhZQk6gfyc81nw?e=iu0nhL)

- [ ] Bug 1: [Accessibility][Programmatic access]: IDs used in ARIA and labels must be unique.
- [ ] Bug 2: [Accessibility][Keyboard Navigation]: Disabled Combo box option 'Option J' is selectable using keyboard.
- [ ] Bug 3: [Accessibility][Functional]: Control 'Scaled example with 1000 items (allowFreeform: T, AutoComplete: T)' is implemented two times on page.
- [ ] Bug 4: [Accessibility][Functional]: On keyboard focus selected value disappears from the editable combo box. 
- [ ] Bug 5: [Accessibility][Functional]: 'Basic uncontrolled example (allowFreeform: T, AutoComplete: T)' combo box saves blank values. 
- [ ] Bug 6: [Accessibility][Functional]: On clicking label 'Scaled example with 1000 items (allowFreeform: T, AutoComplete: T):' using mouse, focus jumps to the 'Basic uncontrolled example (allowFreeform: T, AutoComplete: T)' edit combo box. 
- [ ] Bug 7: [Accessibility][Keyboard navigation]: User have to re-navigate the checkbox options to uncheck/check them at the same time. 
- [ ] Bug 8: [Accessibility][Keyboard Navigation]: Keyboard focus is not visible on 'Basic uncontrolled multi select example (allowFreeform: T, AutoComplete: T)' combobox list options. 
- [ ] Bug 9: [Accessibility][Keyboard Navigation]: When user selects 'Basic uncontrolled example' combo box, keyboard focus does not jump to the first list option.
- [ ] Bug 10:[Accessibility][Keyboard Navigation]: Keyboard tab focus goes to the disabled element inside 'Disabled uncontrolled example with defaultSelectedKey' combobox. 
- [ ] Bug 11:[Accessibility][Functional]: Even when 'allowfreeform' property is true still user input is bound to provided options in multi select combo box using mouse.
- [ ] Bug 12:[Accessibility][Keyboard navigation]: Arrow keys navigation is not working properly on 'Basic uncontrolled example (allowFreeform: F, AutoComplete: F)' combo box. 
- [ ] Bug 13:[Accessibility][Keyboard Navigation]: List options gets scroll up randomly when user navigates on last list option of the 'Scaled example with 1000 items (allowFreeform: T, AutoComplete: T)' combo box in back navigation.
- [ ] Bug 14:[Accessibility][Programmatic access]: Narrator speaks information of the 'Basic uncontrolled example (allowFreeform: F, AutoComplete: F)' combo box twice. 
- [ ] Bug 15:[Accessibility][Keyboard navigation]: 
'Home' and 'End' key does not take the focus to first / last option of the combo box.

- [ ] Bug 16:[Accessibility][Usable]: After selecting 'Set Focus' button, narrator focus does not jump to the 'Basic uncontrolled example (allowFreeform: T, AutoComplete: T):' combo box.
- [ ] Bug 17:[Accessibility][Usable]: Screen gets scroll up when user navigate on 'Basic uncontrolled multi select example (allowFreeform: T, AutoComplete: T)' combo box in scan mode.
- [ ] Bug 18:[Accessibility][Usable]: Narrator doesn't narrates the position number of element "Option d" without expanding it inside Basic uncontrolled example (allowFreeform: T, AutoComplete: T).
- [ ] Bug 19:[Accessibility][Usable]: Narrator doesn't narrates the state 'Collapsed' when user presses Esc on Basic uncontrolled multi select example (allowFreeform: T, AutoComplete: T) combobox to close it.

Component: ComboBox
@micahgodbolt

**Comment by micahgodbolt** [comment]
@jspurlin left some notes in the linked excel file. Check into those before attempting any triage here.

**Comment by micahgodbolt** [comment]
@jspurlin have you had a chance to look over these? David mentioned that you might have put together an email response already?

**Comment by jspurlin** [comment]
@micahgodbolt No, I have not yet had a chance, but I've added it to my backlog and will take a look soon

**Comment by micahgodbolt** [comment]
awesome, thanks!

**Comment by jspurlin** [comment]
@micahgodbolt Ah yes, I have taken a look at these already. I'll add you to the thread

**Comment by dzearing** [comment]
@jspurlin Are there any actionable issues here to take? If not we need to close these.

**Comment by jspurlin** [comment]
@dzearing this isn't very actionable. I'm not sure which items still need to be looked at OR the details about each bug...
